dolphinscheduler-ui/src/js/conf/home/store/dag/actions.js

```diff
@@ -176,8 +176,8 @@ export default {
   copyProcess ({ state }, payload) {
     return new Promise((resolve, reject) => {
       io.post(`projects/${state.projectCode}/process-definition/batch-copy`, {
-        processDefinitionIds: payload.processDefinitionIds,
-        targetProjectId: payload.targetProjectId
+        codes: payload.processDefinitionIds,
+        targetProjectCode: payload.targetProjectId
```

Comment:
I think Id and code have different meanings. The value of ID cannot be given to code